app/controllers/assessments_controller.rb

```diff
@@ -175,6 +176,31 @@ def update
     redirect_to(patient_assessments_url) && return
   end
 
+  def table
```

Comment:
It would make more sense to call this `index`.

Reply:
Updated since changing request to a GET to use `index` now as well: https://github.com/SaraAlert/SaraAlert/pull/604/commits/30d4fbeb075c861ae8a66567d663f31a9b23ee50